components/alerts/alerts.css

```diff
@@ -0,0 +1,47 @@
+.alert-box{
+    box-sizing: border-box;
+    width: 100%;
+    font-size: 2.5rem;
+    padding: 1.3rem;
+    border-radius: 0.4rem;
+    margin: 0.7rem auto;
+}
+.alert-box-border{
+    border: 0.3rem solid black;
+}
+.alert-primary{
+    background-color: rgb(177, 237, 241);
+    color: rgb(42, 139, 219);
+}
+.alert-secondary{
+    background-color: rgb(210, 238, 235);
+    color: rgb(71, 62, 62);
+}
+.alert-success{
```

Comment:
Give atleast a line break before starting new Class styling.

Reply:
ok, Thank you